qiskit/utils/mitigation/_filters.py

```diff
@@ -60,6 +60,11 @@ def cal_matrix(self):
         """Return cal_matrix."""
         return self._cal_matrix
 
+    @cal_matrix.setter
+    def cal_matrix(self, new_cal_matrix):
```

Comment:
Is there a reason why this method definition is moved here? Looks like a merge conflict due to an old code to me.

Reply:
The reason is this: https://github.com/python/mypy/issues/1465
It may be better just to leave a comment since its a mypy bug, but the change is cheap, so I just went for it (admittedly, I did it before I found out it was a bug)